docker/run.sh

```diff
@@ -165,9 +210,10 @@ cat > "$filename".manifest <<-EOF
 }
 EOF
 
-echo "adding $filename.manifest to bucket $BUCKET_NAME"
-gsutil -q cp "$filename".manifest gs://"$BUCKET_NAME"
+  echo "adding $filename.manifest to bucket $DATAEXPORT_BUCKET_NAME"
+  gsutil -q cp "$filename".manifest gs://"$DATAEXPORT_BUCKET_NAME"
 
-# cleanup files
-rm $filename
-rm "$filename".manifest
+  # cleanup files
```

Comment:
A cleanup is for ~life~ all code paths, not just for ~Chiristmas~ DAP data exports